### ISSUE TITLE
Trying to run Coffee in Ubuntu 22.10

### ISSUE THREAD

**Issue by Shamrock70** [nick92/coffee#39]
Hi Mr. Wilkins,
I tried entering the scripts from the Coffee website into the Terminal and it says the repository(" com.github.nick92.coffee") doesn't exist. I downloaded the CODE from Github and it still doesn't load the repository. I read the README file. Is Ninja supposed to be installed separately? I installed the Ninja binary.

**Comment by nick92** [comment]
Yes ninja can be downloaded and installed on ubuntu, if you try and run meson build --prefix=/usr in the root directory of the project and let me know the error i can try and help

**Comment by Shamrock70** [comment]
Hi Mr. Wilkins,

Here's the response i get from the terminal

***@***.***:~/Documents/Coffee$ sudo
add-apt-repository ppa:coffee-team/coffee && sudo apt update
[sudo] password for leprechaun:
Repository: 'deb https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu/
kinetic main'
Description:
Keep up-to-date with current news and weather with Coffee
More info: https://launchpad.net/~coffee-team/+archive/ubuntu/coffee
Adding repository.
Press [ENTER] to continue or Ctrl-c to cancel.
Found existing deb entry in
/etc/apt/sources.list.d/coffee-team-ubuntu-coffee-kinetic.list
Adding deb entry to
/etc/apt/sources.list.d/coffee-team-ubuntu-coffee-kinetic.list
Found existing deb-src entry in
/etc/apt/sources.list.d/coffee-team-ubuntu-coffee-kinetic.list
Adding disabled deb-src entry to
/etc/apt/sources.list.d/coffee-team-ubuntu-coffee-kinetic.list
Adding key to /etc/apt/trusted.gpg.d/coffee-team-ubuntu-coffee.gpg with
fingerprint 0B129FAD570DCBC5A33F7E70E24BA5B6F67B4FFC
Hit:1 http://archive.ubuntu.com/ubuntu kinetic InRelease
Hit:2 http://security.ubuntu.com/ubuntu kinetic-security InRelease

Hit:3 http://ca.archive.ubuntu.com/ubuntu kinetic InRelease

Hit:4 http://ca.archive.ubuntu.com/ubuntu kinetic-updates InRelease

Hit:5 http://ca.archive.ubuntu.com/ubuntu kinetic-backports InRelease
Ign:6 https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu kinetic
InRelease
Hit:7 https://ppa.launchpadcontent.net/git-core/ppa/ubuntu kinetic InRelease
Ign:8 https://ppa.launchpadcontent.net/oguzhaninan/stacer/ubuntu kinetic
InRelease
Err:9 https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu kinetic
Release
  404  Not Found [IP: 185.125.190.52 443]
Err:10 https://ppa.launchpadcontent.net/oguzhaninan/stacer/ubuntu kinetic
Release
  404  Not Found [IP: 185.125.190.52 443]
Reading package lists... Done
E: The repository '
https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu kinetic Release'
does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore
disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration
details.
E: The repository '
https://ppa.launchpadcontent.net/oguzhaninan/stacer/ubuntu kinetic Release'
does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore
disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration
details.
Hit:1 http://archive.ubuntu.com/ubuntu kinetic InRelease
Hit:2 http://security.ubuntu.com/ubuntu kinetic-security InRelease

Hit:3 http://ca.archive.ubuntu.com/ubuntu kinetic InRelease

Hit:4 http://ca.archive.ubuntu.com/ubuntu kinetic-updates InRelease

Hit:5 http://ca.archive.ubuntu.com/ubuntu kinetic-backports InRelease

Ign:6 https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu kinetic
InRelease
Hit:7 https://ppa.launchpadcontent.net/git-core/ppa/ubuntu kinetic InRelease
Ign:8 https://ppa.launchpadcontent.net/oguzhaninan/stacer/ubuntu kinetic
InRelease
Err:9 https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu kinetic
Release
  404  Not Found [IP: 185.125.190.52 443]
Err:10 https://ppa.launchpadcontent.net/oguzhaninan/stacer/ubuntu kinetic
Release
  404  Not Found [IP: 185.125.190.52 443]
Reading package lists... Done
E: The repository '
https://ppa.launchpadcontent.net/coffee-team/coffee/ubuntu kinetic Release'
does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore
disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration
details.
E: The repository '
https://ppa.launchpadcontent.net/oguzhaninan/stacer/ubuntu kinetic Release'
does not have a Release file.
N: Updating from such a repository can't be done securely, and is therefore
disabled by default.
N: See apt-secure(8) manpage for repository creation and user configuration
details.
***@***.***:~/Documents/Coffee$ sudo apt install
com.github.nick92.coffee
Reading package lists... Done
Building dependency tree... Done
Reading state information... Done
E: Unable to locate package com.github.nick92.coffee
E: Couldn't find any package by glob 'com.github.nick92.coffee'

Thanks

Shawn

**Comment by Shamrock70** [comment]
HI Mr. Wilkins,

I entered:
1. sudo add-apt-repository ppa:coffee-team/coffee
2. sudo apt update
3. sudo apt install com.github.nick92.coffee
4. com.github.nick92.coffee

[image: image.png]

The app will open and I'll choose a news feed and then the linux wheel will
appear next to "Launch Coffee" and begin to spin and nothing happens.

Thanks

Shawn

**Comment by Shamrock70** [comment]
I installed the ubuntu (2.2.0) 19.04 version of the coffee weather app.
